src/site/assets/js/medallia/vagovprod.js

```diff
@@ -0,0 +1,48 @@
+// Medallia Embed Script
+(function(g) {
+  var isStaging = window.location.host.includes('staging');
+  if (!isStaging) return
```

Comment:
What if it's dev or localhost - is this okay to have execute?

Reply:
I think we only want the survey on staging. Eventually, we will want it on prod. 

For example, a different Medallia survey gets loaded on staging.va.gov/find-locations, but not dev.va.gov/find-locations.

![image](https://user-images.githubusercontent.com/6130520/106619184-ec79a800-6535-11eb-9f29-405c8d4e1789.png)

Reply:
Oh yeah I read this wrong. Makes sense. Thanks!